CHANGELOG.rst

```diff
@@ -17,6 +17,9 @@ Changes
 Added Features
 --------------
 
+* Added a setting to disable the notification stream. This may improve Girder's performance runtime environments with a
```

Comment:
`Added Features` -> `New Features`

Reply:
Sorry, I may have been unclear.

I meant that in a section called `Added Features` I don't like saying `Added a setting` but prefer `Setting to disable`.

Again, this is not something that anyone should bother paying attention to as it is really unimportant.

Reply:
@mgrauer Apparently I forgot how to use prepositions, so I'll fix the missing *to*.

However, since a number of threads is a countable quantity, not a matter of size or degree, the adjective *fewer* should be used: http://www.gmatpill.com/more-than-vs-greater-than-vs-less-than-fewer-than/

Thanks for the feedback. For many developers, the CHANGELOG is the primary feed of Girder updates, so I'm happy to spend time making it polished and easy to follow.

Reply:
Right, it would be `fewer threads` because threads are countable [1] but you said `a fewer number of threads`. It should be either a `smaller number of threads` or `fewer threads`.

And it turns out I may be wrong about `smaller number`, it could end up being `lower number` or `lesser number`, see [here](https://english.stackexchange.com/questions/26471/lower-number-vs-smaller-number). As English is descriptive rather than prescriptive, perhaps that blue line in the top chart wins the day and it is `smaller number` after all? Maybe `fewer threads` is best as it sidesteps this sticky wicket.

🥀 

How's that for burnishing the bikeshed!

[1] good thing we are not using greenlets which are uncountable [2]
[2] joking